### PR TITLE
Add info alerts for states when feedback isnt available

### PR DESCRIPTION
**Pull request by bethanyaconnor**
When the level isn't the project level:

Screenshot 2023-10-19 at 7 30 10 AM

When no student is selected:

Screenshot 2023-10-19 at 7 30 40 AM

Look and working copied from the [figma](https://www.figma.com/file/WBcs7BY9IC6Nq5rG4V1SWF/AI-in-Teacher-Tools?node-id=2946%3A5612&mode=dev). My understanding is that this banner will eventually be a part of the component library but hasn't been built yet.

### CODE REVIEW

File: apps/src/templates/rubrics/RubricContainer.story.jsx [hunk 1]
```diff
@@ -91,6 +91,7 @@ const Template = args => (
     teacherHasEnabledAi={false}
     studentLevelInfo={defaultStudentLevelInfo}
     currentLevelName={rubricLevelName}
+    open
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/RubricContainer.story.jsx:94]: I should've set this prop in https://github.com/code-dot-org/code-dot-org/pull/54153.

File: apps/src/templates/rubrics/RubricContent.jsx [hunk 2]
```diff
@@ -118,13 +119,20 @@ export default function RubricContent({
     }
   };
 
+  let infoText = null;
+  if (!onLevelForEvaluation) {
+    infoText = i18n.rubricCanOnlyBeEvaluatedOnProjectLevelAlert();
+  } else if (!studentLevelInfo) {
+    infoText = infoText = i18n.selectAStudentToEvaluateAlert();
```
Comment (Nokondi) [inline apps/src/templates/rubrics/RubricContent.jsx]: Did you mean to assign infoText twice in this line?

**Review by levadadenys — APPROVED** [review]
LGTM!
one nit: looks like font awesome icon should also have a `light-info-500` color instead of black.

Thanks for tagging me in! Yeah, this info alert will be part of Phase 2 Design System component. In phase 2 we'll call it `Notification Banner / Alerts` probably and designs are I guess 80-90% ready. Once front end will get there, we'll probably use this as a base for the component. (created a ticket for that and added a link to this PR so we won't forget about it).

File: apps/src/templates/rubrics/RubricContent.jsx (hunk 2 above)
Comment (bethanyaconnor) [reply apps/src/templates/rubrics/RubricContent.jsx]: 🤦‍♀️ nope lol

**Review by Nokondi — APPROVED** [review]

**Pull request merged**